vuepress/docs/next/tutorials/consume/solr.md

```diff
@@ -75,13 +75,14 @@ kubectl scale deploy/YOUR-APP-NAME-deployment --replicas=0 -n YOUR-NAMESPACE
 
 2. Edit the deployment and add the following environment variables:
 
-``` bash
+``` yaml
 spec
-  -env
-     - name: SOLR_ACTIVE
-       value: "true"
-     - name: SOLR_ADDRESS
-	     value: http://solr-solrcloud-common/solr 
+   containers:
+     - env
+       - name: SOLR_ACTIVE
+         value: "true"
+       - name: SOLR_ADDRESS
+	       value: http://solr-solrcloud-common/solr 
```

Comment:
Indentation looks off here, based on the git diff there's some odd characters. The name and value lines should start at the same column,

Reply:
oddly, it looks fine in vs code.